src/main/resources/config.yml

```diff
@@ -15,7 +15,7 @@ general:
   # If there is no economy plugin present anyway, money will be automatically disabled.
   use-economy: true
   database:
-    # YAML, JSON, MYSQL, MONGODB.
+    # YAML, JSON, MYSQL, MARIADB, MONGODB.
```

Comment:
That'll be removed when the config will be regenerated when releasing 1.1. Edit the `@ConfigComment` value in the Settings class instead.